packages/v2/gui/src/shared/InteractiveList/InteractiveList.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import classnames from 'classnames';
+
+import { ChevronDownIcon, ChevronRightIcon } from '@navikt/aksel-icons';
+
+import styles from './interactiveList.module.css';
+
+export interface InteractiveListElement {
+  content: React.ReactNode;
+  onClick: () => void;
+  active: boolean;
+}
+
+interface InteractiveListProps {
+  elements: Array<InteractiveListElement & { key?: string }>;
+}
+
+const InteractiveListElement = (props: InteractiveListElement) => {
+  const { content, active, onClick } = props;
+  const cls = classnames(styles['interactiveListElement'], {
```

Comment:
Foreslår å droppe classnames funksjon og bruke dot annotasjon for styles, slik:

```suggestion
 const cls = `${styles.interactiveListElement} ${active ? styles.interactiveListElementActive : styles.interactiveListElementInactive}`;
```